content/docs/reference/programming-model.md

```diff
@@ -115,27 +115,36 @@ The `args` provided to a resource determine what inputs will be used to initiali
 
 All resource constructors also accept an `options` argument which can provide the following additional resource options controlling how the resource will be managed by Pulumi.
 
+###### `additionalSecretOutputs`
+Provides a list of output properties which should be treated as secrets. This value augments any values that Pulumi detects itself, based on what secret inputs to the resource has. This is typically used to express that for a specific instance of a resource, some of its output properties should be treated as secrets (when they would not normally be).
+
+###### `aliases`
+Lets you rename a resource. When making a breaking change to the name or type of a resource or component, you can add the old name to the list of `aliases` for a resource to ensure that existing resources will be migrated to the new name instead of being deleted and replaced with the new named resource.
+
+###### `customTimeouts`
```

Comment:
@stack72 I added an example here, but none of the other resource options have them.

Reply:
Here's what I had written locally:

> Provides a set of custom timeouts for a resource's create, update, and delete operations. These timeouts are specified as a string like "5m", "40s", or "1d" (5 minutes, 40 seconds, or 1 day, respectively).

I do believe that the formatting examples are particularly important here, though they are also present in the language SDKs.

Reply:
Thanks, @pgavlin . If these are options that a user would set or pass in, I do wonder about the usage of "Provides a ...". Please educate me if I am completely off mark. For now, I think we should try to be consistent so I updated the `aliases` description as well to follow the current description pattern.